platform/src/test/java/platformfeatures/PF_Footer.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Validate footer links
+@FooterPage
+Scenario Outline: Validate Footer Links
+
+Given Navigate to URL and check footer links <browser>
```

Comment:
Please break open browser into seperate step
Ex: Given Navigate to URL <browser>
When User clicks on footer links

Reply:
Done changes as requested

Reply:
yup. We will edit it after merging to work on new runner